main.cpp

```diff
@@ -0,0 +1,19 @@
+//  Matt Perry, Tailon Russell, Chenyi Ling, McKade Sorensen
+
+#include <iostream>
+#include "shape.hpp"
+#define CATCH_CONFIG_RUNNER
+#define CATCH_CONFIG_FAST_COMPILE
+#include "catch.hpp"
+
+int main(int argc, char* argv[]) {
+    Catch::Session testing;
+    int catchresult;
+    catchresult = testing.applyCommandLine(argc, argv);
+    if (!catchresult) {
+        std::cout << "Begin tests for class shapes and its derived classes: \n";
+        catchresult = testing.run();
+        std::cout << "End of automated tests.\n"; 
+    }
+    return 0;
+}
```

Comment:
Not something to be fixed but new lines at the end of a file are good practice for whatever reason (I should look up why)

Reply:
like return 0; newline newline?

Reply:
![Screen Shot 2020-04-01 at 3 06 48 PM](https://user-images.githubusercontent.com/49543713/78194736-721a5c00-742a-11ea-8696-cf72e47ca566.png)

Reply:
No after the last bracket in this case